winterfell/forms/VBA-21-0966-ARE.json

```diff
@@ -1,4 +1,8 @@
 {
+  "unofficialTitle": "Intent to File",
+  "unofficialDescription": "This form is used to notify VA of your intent to file for benefits. Even if your claim fails to process, your benefits effective date will still be the date the VA receives this form.",
+  "required": true,
```

Comment:
This form is required for submitting a valid claim. If you do not complete this/these required form(s), you cannot submit a claim.